CMake/cdat_modules/singledispatch_pkg.cmake

```diff
@@ -0,0 +1,16 @@
+set( SINGLEDISPATCH_MAJOR 3 )
```

Comment:
setup.py on ubuntu lists it as a dependeny. So inside our firewall it died because it couldn't fetch it. It's yet another example of the "cool/sneaky" features of pip...